docs/mathematics/catalan-number.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Catalan number
+slug: /mathematics/catalan-number
+---
+
+## Introduction
+
+Catalan numbers are a sequence of natural numbers that occur in various counting problems. The solution to many counting problems in combinatorial mathematics is given by Catalan numbers.
```

Comment:
Can you share a bit more description? It seems too vague to me.

Reply:
I searched it online. There are its applications, but nothing more to explain.